converters/google/resources/org_policy_policy.go

```diff
@@ -0,0 +1,184 @@
+package google
+
+import (
+	"fmt"
+	"strings"
+)
+
+func resourceConverterOrgPolicyPolicy() ResourceConverter {
+	return ResourceConverter{
+		Convert:           GetOrgPolicyPolicyCaiObject,
+		MergeCreateUpdate: MergeOrgPolicyPolicy,
+	}
+}
+
+func GetOrgPolicyPolicyCaiObject(d TerraformResourceData, config *Config) ([]Asset, error) {
+
```

Comment:
It does require additional changes in config-validator. Raised the PR in https://github.com/GoogleCloudPlatform/config-validator/pull/199

Reply:
ah, I was only searching for the field names, not the Getters! 💯

Reply:
@jashan-g it looks like the issue is https://github.com/GoogleCloudPlatform/config-validator/blob/a4226f60974b18f4ed03995b0c9b5463dc39a3b2/pkg/gcptarget/gcptarget.go#L186 - it needs to also check for v2 org policies - something like the following seems to resolve the issue for me locally:

```
foundV2OrgPolicy := false
if asset["v2_org_policies"] != nil {
	foundV2OrgPolicy = true
}
```

You'll also need to add a block to the following resourceTypes check.

It would also be great to add tests to `TestReviewAsset` to check the behavior of at least the v2 org policy case. It would be nice to also get some base cases in there for the various types of resources if you have time, since it doesn't look like we were testing this previously - but just the one case is fine otherwise.

Reply:
Addressed the issue in https://github.com/GoogleCloudPlatform/config-validator/pull/200/files
Also added tests to TestReviewAsset for v2 org policy and other types of resources.